scintellometry/io/filehandlers.py

```diff
@@ -290,7 +290,7 @@ def read(self, size):
         while(iz < size):
             self._seek(self.offset)
             block, already_read = divmod(self.offset, self.filesize)
-            fh_size = min(size - iz, self.filesize - already_read)
+            fh_size = int(min(size - iz, self.filesize - already_read)) #Rob added, cast to int
```

Comment:
Why should this ever be necessary? It would only happen if self.filesize is not an integer, which would seem a mistake.